video/src/ai/eto/rikai/sql/spark/datasources/VideoOptions.scala

```diff
@@ -1,21 +1,65 @@
 package ai.eto.rikai.sql.spark.datasources
```

Comment:
My IDE say it `Package name 'ai.eto.rikai.sql.spark.datasources' does not correspond to the file path '' `, maybe it's legal in mill, but still confusing readers, how about changing the file position to make it align with package name.